djoser/views/user/user.py

```diff
@@ -0,0 +1,92 @@
+from django.contrib.auth import get_user_model
+from rest_framework import status, viewsets
+from rest_framework.exceptions import NotFound
+from rest_framework.response import Response
+
+from djoser import signals, utils
+from djoser.conf import settings
+from djoser.compat import get_user_email
+from django.contrib.auth.tokens import default_token_generator
+
+User = get_user_model()
+
+
+class UserViewSet(viewsets.ModelViewSet):
```

Comment:
should it be split further?

Reply:
I think it makes sense to split completely small views.

I think the whole point of splitting is that you can easily get any view, extend it or modify its behavior and inject to your project ONLY what you want.

With this viewset if you don't want to have e.g. update code in you project you can only remove the url pattern, but not the view handler.

Taking a lesson from the community feedback, I would say that viewset is not the best choice for a library.

Reply:
I split everything as much as possible, but I doubt `me` can be split further as then we can't register it under the same endpoint, even though it uses a different http method, no?